projects/plugins/boost/app/assets/src/js/pages/settings/sections/Score.svelte

```diff
@@ -95,15 +178,38 @@
 				<MobileIcon />
 				<div>{__( 'Mobile score', 'jetpack-boost' )}</div>
 			</div>
-			<ScoreBar bind:score={scores.mobile} active={siteIsOnline} {isLoading} />
+			<ScoreBar
+				bind:prevScore={scores.previous.mobile}
+				bind:score={scores.current.mobile}
+				active={siteIsOnline}
+				{isLoading}
+				{showPrevScores}
```

Comment:
I have removed the `bind:` because we do not need two way data flow here. `prevScore` can benifit from optional chaining, which is not possible with binding.